core/src/main/scala/cats/Reducible.scala

```diff
@@ -227,7 +227,7 @@ abstract class NonEmptyReducible[F[_], G[_]](implicit G: Foldable[G]) extends Re
   }
 
   override def get[A](fa: F[A])(idx: Long): Option[A] =
-    if (idx == 0L) Some(split(fa)._1) else G.get(split(fa)._2)(idx)
+    if (idx == 0L) Some(split(fa)._1) else G.get(split(fa)._2)(idx - 1L)
```

Comment:
this is not actually tested according to codecov.

Can we write a test to explicitly exercise this?